docs/program/apis/sessions.md

```diff
@@ -0,0 +1,88 @@
+A session technically is any presence mechanism at the RDK application layer maintained by any client with `viam-server`.
+Session management, as explained here, allows for safer robotic operation when you are dealing with moving parts on the robot (the "actuating controls").
+
+Specifically, without session management, controls that would be "sticky" (for example [`SetPower()` of a base](/components/base/#setpower)) based on the last input of a **client**, meaning the API request from the client sets the flow of electricity on a part of the robot and then does not time out, could lead to **having a robot trying to continue doing what it was told to do forever.**
+
+You do not want that to happen.
+
+#### Clients and Viam's Client SDKs
+
+A client has multiple meanings for a Viam robot.
+Essentially, it's anything that is receiving the information served by `viam-server` running on the robot.
+
+A **client** can be an SDK script controlling the robot, an input controller, or just the different resources on the robot talking amongst themselves.
+For example, if you use Viam's module registry to [add modular resources to your robot](/extend/modular-resources/), the clients of your robot in its lifetime will include the "model servers" you instantiate on your robot for individual resources.
+
+Viam's Session Management API is your built-in solution to this.
+Your client maintains the session, telling the `viam-server` instance that it is still present every so often, or staying within the heartbeat window.
+
+Your client must send at least one session heartbeat within this window.
+As soon as the window lapses/expires, the server will safely stop all resources that are marked for safety monitoring that have been last used by that session, and no others; a lapsed client will attempt to establish a new session immediately prior to the next operation it performs.
+
+{{< /alert >}}
+
+## Usage
+
+Usage of the Sessions Management API differs across [Viam's SDKS](/program/).
+
+{{< tabs >}}
+{{% tab name="Go" %}}
+
+### On-Robot Session Management
+
+To use the [Session Management API](https://pkg.go.dev/go.viam.com/rdk/session) the Go Client SDK provides, use your [`RobotClient`](/program/apis/#robot-api) instance (client of the Robot API) to instantiate a `SessionsManager` with `SessionManager()`.
+
+### Sessions for RobotClients
+
+First, use your [`RobotClient()`](/program/apis/#robot-api) instance to access the [`SessionsClient`](https://pkg.go.dev/go.viam.com/rdk/session) within your Go Client SDK program.
+This is a [gRPC](https://grpc.io/) client that `viam-server` instantiates at robot runtime.
+Find `SessionsClient` defined on [Github](https://github.com/viamrobotics/rdk/blob/main/robot/client/client.go).
+
+{{% /tab %}}
+{{% tab name="Python" %}}
+
+### On-Robot Session Management
+
+The `Session Management API` is not currently provided in the Python SDK.
+Use the Go Client SDK instead.
+
+### Sessions for RobotClients
```

Comment:
```suggestion
### Manage sessions for robot clients
```
or something more active

Reply:
done

Reply:
Unsure about removing `RobotClient` but I think "robot clients" is probably fine

Reply:
Either way